controllers/FilesController.js

```diff
@@ -0,0 +1,48 @@
+import dbClient from "../utils/db";
+import { v4 as uuid4 } from "uuid";
+import redisClient from "../utils/redis";
+const fs = require('fs');
```

Comment:
use es module. it should be `import 'fs'`

Reply:
thats true, thanks for the heads-up